config/config.example.yml

```diff
@@ -137,6 +139,16 @@ https_only: false
 ##
 ## Enable/Disable the use of QUIC (HTTP/3) when connecting
 ## to the youtube API and websites ('youtube.com', 'ytimg.com').
+## QUIC's main advantages are its lower latency and lower bandwidth
+## use, compared to its predecessors. However, the current version
+## of QUIC used in invidious is still based on the IETF draft 31,
+## meaning that the underlying library may still not be fully
+## optimized. You can read more about QUIC at the link below:
+## https://datatracker.ietf.org/doc/html/draft-ietf-quic-transport-31
+##
+## Note: you should try both options and see what is the best for
```

Comment:
`## Note: you should try both options and see what is the best for your instance, but in general QUIC is recommended on public instances, and not recommended for private one. Your mileage may vary.`

Reply:
I've always had QuIC enabled on my private instance. It doesn't seem to be an issue...

Reply:
>It doesn't seem to be an issue...

From what I know it's doesn't cause any issue, but I heard that it's where the memory leaks come from, so avoid it is preferable if it's possible. As I said, I haven't done any testing.

Reply:
Disabling QUIC tend to reduce the frequency at which you are going to restart the instance, but it does not solve the memory or crashing issues.
In the end you will just have to restart Invidious less frequently during the day.

Reply:
Should I keep the "Some maintainers reported that using QUIC may reduce the occurence of captcha challenges requested by youtube.", part?

Reply:
@SamantazFox If it's true... sure. I have no idea if this is though (ping @unixfox )

Reply:
> Should I keep the "Some maintainers reported that using QUIC may reduce the occurence of captcha challenges requested by youtube.", part?

There are two CAPTCHA like explained here: https://github.com/iv-org/invidious/issues/957#issuecomment-576424042. Invidious just bypass the first one when QUIC is being used, it doesn't reduce the occurrence of CAPTCHA challenges for the second CAPTCHA.

Reply:
ok. I'll reword the thing and add a link to that specific message.